client/components/ProgressBar.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+// import { statement } from '@babel/template'
```

Comment:
Try not to include commented out code in your commits. If it's not being used, there's probably no need for it to be here :-)

Reply:
Comment removed